index.html

```diff
@@ -11,6 +11,26 @@
 
 </head>
 
+<header>
+    <section class="top-header">
+        <div class="nav-header">
+            <div class="lambdapng">
+                <img src="img/lambda-black.png" alt="Lambda">
+            </div>
```

Comment:
One more thing to do here is to remove the text-decoration from the nav links like in the design file.